wporg-gp-translation-events.php

```diff
@@ -162,12 +161,47 @@ public function event_meta_boxes() {
 	 */
 	public function event_dates_meta_box( WP_Post $post ) {
 		wp_nonce_field( 'event_dates_nonce', 'event_dates_nonce' );
-		$event_start = get_post_meta( $post->ID, '_event_start', true );
-		$event_end   = get_post_meta( $post->ID, '_event_end', true );
-		echo '<label for="event_start">Start Date: </label>';
-		echo '<input type="date" id="event_start" name="event_start" value="' . esc_attr( $event_start ) . '" required>';
-		echo '<label for="event_end">End Date: </label>';
-		echo '<input type="date" id="event_end" name="event_end" value="' . esc_attr( $event_end ) . '" required>';
+		$event_start    = get_post_meta( $post->ID, '_event_start', true );
+		$event_end      = get_post_meta( $post->ID, '_event_end', true );
+		$event_timezone = get_post_meta( $post->ID, '_event_timezone', true );
+		$hosts          = explode( ',', get_post_meta( $post->ID, '_hosts', true ) );
+		if ( empty( $hosts ) ) {
```

Comment:
If `_hosts` is an empty value, the `$host` value will be `array(  0 => "")`, so we need to check the first value in the array: 

```suggestion
		if ( empty( $hosts[ 0 ] ) ) {
```